src/ICat.hs

```diff
@@ -350,22 +77,25 @@ handlePicture (Right picture) =
             Nothing ->
               4
       -- Print out the picture.
-      Control.Monad.forM_ [0, step .. ph - 1] $
+      forM_ [0, step .. ph - 1]
         (\y ->
           do
-            Control.Monad.forM_ [0, step .. pw - 1 ] $
+            forM_ [0, step .. pw - 1 ]
               (\x ->
-                do 
-                  IO.hPutStr IO.stdout $ escapeCodeAt pictureRGB8 x y
-                  IO.hPutStr IO.stdout " ")
-            IO.hPutStrLn IO.stdout "\ESC[0m")
+                do
+                  -- everybody knows where these come from
+                  -- also hPutStr stdout == putStr
```

Comment:
I was originally using `putStr` but I seem to remember there being a problem with that and ghci in Emacs; nothing would print out. It might have just been an artifact of a long running session though.